nodes/src/nodes/prompt/README.md

```
@@ -0,0 +1,69 @@
+
+## Configuration
+
+| Field        | Description                                                                       |
+| ------------ | --------------------------------------------------------------------------------- |
+| Instructions | One or more instruction strings added to `### System Instructions:` in the prompt |
+
+## Rendered prompt structure
+
+When the Question is consumed by an LLM or agent, it is rendered in this order:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)